chainextensions/dapps-staking/traits/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "chain-extension-traits"
+version = "4.0.0"
```

Comment:
Two questions:
* one repeated but why 4.0.0?
* `chain-extension-traits` seem to be very generic but they're under _chainextension/dapps-staking_. Why?

Reply:
Yes the Trait will be used by any impls of chain-extensions
I refactored the folder structure to meet future implementations